### DISCUSSION TITLE
Inserting an image into a post and displaying it?

### DISCUSSION THREAD

**Discussion by escfrpls** [community/community#121828] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello!

Can you guys tell me how to fix the error?
I inserted an image link into my post.

`![Image](https://github.com/escfrpls/drochiczynpoleski/blob/2fa0d0ecdf4ebf2aab1dc81ecbca68f6c76daf3e/drgpls/content/img/Wawulicze/1725_53659884938_o.jpg)`

My page is deploying on netlify and when everything is published the picture doesn't show up.

![Screenshot_20240501_171548](https://github.com/community/community/assets/125834172/7b204ee3-ab9c-4a66-8587-72d0cd7de26e)

p.s.
The GitHub post itself displays it
![image](https://github.com/community/community/assets/125834172/a953a00b-d06f-455d-839d-46bd9bbb25ff)

**Comment by dane** [comment]
Can you share the image tag that Netlify generated or a link to the page with the broken image?

**Comment by escfrpls** [comment]
fixed
